docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,104 @@
+# Convert ONNX* RNNT to the Intermediate Representation (IR) {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction covers conversion of RNN-T model from [MLCommons](https://github.com/mlcommons) repository. Follow 
+the steps below to export a PyTorch* model into ONNX* before converting it to IR:
```

Comment:
PyTorch* model into ONNX* ---> PyTorch\* model into ONNX\* 
(to avoid italics)

Reply:
done, added escape special character `\` for all asterisks